src/cloud/paths/tasks.yml

```diff
@@ -212,7 +212,9 @@ post:
     content:
       application/json:
         schema:
-          $ref: "../schemas/TaskCreateRequest.yml"
+          oneOf:
```

Comment:
@asalem1 @hoorayimhelping This PR splits the `POST /tasks` request schema into `flux` vs `scriptID` request schemas. Any idea if this `oneOf` will cause a problem for the UI or oats clients? Obviously, it's valid, but I don't want to break implementations that don't support it.

Reply:
@jstirnaman I don't think `oneOf` will cause problems in and of itself. I see some other examples of it being used in openapi code the UI uses. The biggest risk is the change itself - it looks like this PR adds `TaskWithScriptCreateRequest.yml` in addition to using `TaskCreateRequest.yml`. Changing the description of these parameters and adding a new one might cause issues that aren't obvious right now

Reply:
@hoorayimhelping Ok. There isn't anything new here, just explicitly defining the way the API works, but, yeah [the test run](src/tasks/containers/TaskPage.test.tsx) that Gene started for it is failing. Not sure why yet.

Reply:
@jstirnaman 

```
    src/tasks/containers/TaskPage.test.tsx:259:66 - error TS2339: Property 'flux' does not exist on type 'TaskCreateRequest | TaskWithScriptCreateRequest'.
      Property 'flux' does not exist on type 'TaskWithScriptCreateRequest'.
```

Reply:
> @jstirnaman
> 
> ```
>     src/tasks/containers/TaskPage.test.tsx:259:66 - error TS2339: Property 'flux' does not exist on type 'TaskCreateRequest | TaskWithScriptCreateRequest'.
>       Property 'flux' does not exist on type 'TaskWithScriptCreateRequest'.
> ```

I'm assuming it needs a discriminator to know which schema to use? I ran `yarn generate` to generate `src/client/*` locally and I don't even see `TaskWithScriptCreateRequest` in the results, so I 'm confused.
I'll undo this for now and move it to my _someday_ list. It seems beneficial to clients if they could discern which properties are compatible and required. I hate having it human-readable-only fields.

Reply:
@sunbryely-influxdata Can you review this for my peace of mind? Because of the failing UI test, I've reverted my change that used `oneOf` for `POST /tasks` schemas in Cloud.